docs/backlog-management.md

```diff
@@ -0,0 +1,79 @@
+# Issue Management for OpenTelemetry
+
+It's an important community goal for OpenTelemetry that our members find the backlogs to be responsive, and easy to take part in. Having a shared methodology for processing issues helps new members understand the current state of each project.
+
+SIGs are encouraged to experiment with labels and backlog management procedures. This document only covers the bare bones of issue management which should work the same across all SIGs, to help maintain a responsive backlog and allow us to track work across all projects in a similar manner.
+
+## New Issues
+
+Flow:
+* An issue is filed by a community member.
+* A maintainer responds and asks clarifying questions.
+* The maintainer processes the issue as a Code Change, a Discussion, or resolves it quickly.
```

Comment:
next sentence saying that it will be a triager, not maintainer role. Which one do you propose?